storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.7.rst

```diff
@@ -0,0 +1,60 @@
+.. _rn.8-0-7:
+
+================================================================================
+|pxb.name| |release|
+================================================================================
+
+|percona| is glad to announce the release of |pxb.name| |release| on |date|.
+Downloads are available from our `download site
+<https://www.percona.com/downloads/Percona-XtraBackup-LATEST/>`_ and from
+:ref:`apt <apt_repo>` and :ref:`yum <yum_repo>` repositories.
+
+|pxb.name| enables MySQL backups without blocking user queries, making it ideal
+for companies with large data sets and mission-critical applications that cannot
+tolerate long periods of downtime. Offered free as an open source solution, it
+drives down backup costs while providing unique features for |MySQL| backups.
+
+In release |release|, |pxb.name| introduces the encryption of the system
```

Comment:
xtrabackup does not introduce "encryption of the system tablespace", it now supports backups and prepare of the databases containing encrypted system tablespace (https://www.percona.com/doc/percona-server/5.7/management/data_at_rest_encryption.html#innodb_sys_tablespace_encrypt)

also, xtrabackup now supports encrypted mysql tablespace as well